### ISSUE TITLE
variable @button-text-color-selected is undefined

### ISSUE THREAD

**Issue by N3stY** [suda/tool-bar#223]
just updated the package to the latest version and there was such an error: **variable @button-text-color-selected is undefined in C:\Users\N3stY\.atom\packages\tool-bar\styles\tool-bar.less:37:13**

Problem with the theme. I use Atom Material but with all the standard themes same problem, except One Dark. 
I have "fixed" it with 
```
@import "ui-variables";
...
@button-text-color-selected: @button-border-color; // FIX

.tool-bar {
...
```

**Comment by edreesen** [comment]
duplicate of #219, solution can be found there

**Comment by suda** [comment]
Duplicate of https://github.com/suda/tool-bar/issues/219. Fixed in 1.1.6
